pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/TopicPolicies.java

```diff
@@ -71,6 +71,10 @@ public class TopicPolicies {
     private Integer maxSubscriptionsPerTopic;
     private DispatchRateImpl replicatorDispatchRate;
     private SchemaCompatibilityStrategy schemaCompatibilityStrategy;
+    /**
+     * Dispatch rate limiter setting for specific subscription.
+     */
+    private Map<String/*subscription*/, DispatchRateImpl> subscriptionLevelDispatchRateMap = new HashMap<>();
```

Comment:
Do we need to introduce SubscriptionPolicies here? We might need to support other policies like backlog quota, TTL.

Reply:
And need to add `@Builder.Default` annotation? details to see https://github.com/apache/pulsar/pull/12009

Reply:
> Do we need to introduce SubscriptionPolicies here? We might need to support other policies like backlog quota, TTL.

Good point.

Reply:
You forget to add the `@Builder.Default` here.